aspnetcore-webapi/RequestBodyRead/RequestBodyRead/Shared/DataTransferObjects/UserForCreationDto.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+namespace RequestBodyRead.Shared.DataTransferObjects
+{
+	public record UserForCreationAndUpdateDto
+	{
+        public string? FirstName { get; set; }
+
+        public string? LastName { get; set; }
+
+        public string? Email { get; set; }
```

Comment:
Instead of defining these as auto-properties, is it possible to make use of the record constructor? So something like:
```csharp
public record UserForCreationAndUpdateDto(string? FirstName = null, string? LastName = null, string? Email = null);
```